docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
+  initialize: function(options) {
+    console.log('Initialize');
+  },
+
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(options.data); 
```

Comment:
Here fixed example https://jsfiddle.net/marionettejs/k05dctyt/. Please, add it instead of yours.